app-text/mandoc/mandoc-1.14.3.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A suite of tools compiling mdoc from the OpenBSD project"
+DESCRIPTION="A suite of tools compiling mdoc, the roff macro language of choice \
+for BSD manual pages, and man, the predominant historical language for UNIX \
+manuals. It is small, ISO C, ISC-licensed, and quite fast. The main component \
+of the toolset is the mandoc utility program, based on the libmandoc validating \
+compiler, to format output for UTF-8 and ASCII UNIX terminals, HTML 5, \
+PostScript, and PDF."
+HOMEPAGE="http://mandoc.bsd.lv/"
+COPYRIGHT="1989-2017 mandoc Authors"
+# Note: update LICENSE on every new version
+LICENSE="mandoc-$portVersion"
```

Comment:
You could simply list the licenses (ISC, BSD 2 and 3 clause).  ISC seems to be missing at https://git.haiku-os.org/haiku/tree/data/system/data/licenses though

Reply:
I'd use this:
```
LICENSE="mandoc-$portVersion
	BSD (2-clause)
	BSD (3-clause)
	ISC"
```

BTW, if a recipe for another release of mandoc is added it will probably have a very similar `LICENSE` file, so maybe you can rename `mandoc-$portVersion` to just `mandoc`. If anything is added to the `LICENSE` file on any future release of mandoc then you would only need to update `app-text/mandoc/licenses/mandoc`.

Regarding the `ISC` license, are there any plans to add it to the pool of licenses shipped in `/system/data/licenses/` by the haiku package?